content/webapp/pages/exhibition-guide.tsx

```diff
@@ -180,12 +181,12 @@ function getTypeTitle(type: GuideType): string {
 export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
   async context => {
     const serverData = await getServerData(context);
-    const { id, type, usingQRCode, userPreferenceSet } = context.query;
+    const { id, type, usingQRCode, userPreferenceSet, stopId } = context.query;
```

Comment:
Where is this being used as a query param (rather than as a fragment)?

Reply:
(Will explain in the PR comment)